exercises/exercise-13/index.ts

```diff
@@ -66,6 +67,12 @@ async function testUsersDatabase() {
     // see more https://docs.mongodb.com/manual/reference/operator/query/lt/
     expect((await usersDatabase.find({age: {$lt: 30}})).map(({_id}) => _id)).to.have.same.members([0, 2, 4, 7]);
 
+    // Can't use any of these operators with array types.
+    expect(
+        // @ts-ignore
+        (await usersDatabase.find({grades: {$eq: 'Magical entity'}})).map(({_id}) => _id)
```

Comment:
Since `grades` is array-valued, this shouldn't be allowed. It was actually a little unclear to me from the problem statement if non-scalars were to be allowed in records, but they're valid JSON, so I didn't see why not.